help: Document how to change the language of the Zulip mobile app.

drrosa opened the pull request:
**Screenshots and screen captures:**

- https://zulip.com/help/change-your-language

![image](https://user-images.githubusercontent.com/2343554/220920863-a911501b-6dd7-484f-9d79-40a7eb822d4d.png)

<details>
<summary>Self-review checklist</summary>

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability

Communicate decisions, questions, and potential concerns.

- [x] Automated tests.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Links.
</details>

drrosa commented:
@alya This PR is ready for you.

alya commented on help/change-your-language.md:
```diff
@@ -19,6 +19,18 @@ messages you receive.
+
+1. Tap **Language**.
+
+1. Use the search tool or scroll through the list of languages.
```
maybe "search bar" rather than "search tool"?

alya commented on help/change-your-language.md:
```diff
@@ -19,6 +19,18 @@ messages you receive.
+1. Use the search tool or scroll through the list of languages.
+
+1. Select a language.
+
```
How about adding an instruction for how to get out of the menu? Something like:

1. Tap the **arrow** (`<icon>`) in the upper left to exit the language selection menu.

alya commented:
Looks good to me other than the above!

drrosa commented:
I updated the PR and the screenshot in response to your feedback.

Also, I noticed a few formatting issues when referring to mobile icons elsewhere so I added another commit to fix them.

alya commented:
Works for me! @timabbott please take a look.

timabbott commented:
Merged, thanks @drrosa!
